tests/plugins/balloontoolbar/positioning.js

```diff
@@ -92,8 +92,9 @@
 		},
 
 		'test panel - out of view - hcenter top': function( editor ) {
-			if ( editor.name == 'divarea' ) {
-				// divarea tests are failing, it's an upstream issue from balloonpanel (#1064).
+			if ( editor.name == 'divarea' || ( bender.config.isTravis && bender.tools.isBuild() ) ) {
+				// divarea tests are failing, it's an upstream issue from balloonpanel (#1064).ga
```

Comment:
```
// divarea tests are failing, it's an upstream issue from balloonpanel (#1064).ga
```

What is this "ga" doing here? 👮

Reply:
It's my shortcut from 'git add'. I probably write it here instead of terminal :(